test/kubemark/resources/hollow-node_template.json

```diff
@@ -106,7 +106,7 @@
 				},
 				{
 					"name": "hollow-proxy",
-					"image": "gcr.io/{{project}}/kubemark:latest",
+					"image": "{{registry}}/{{project}}/kubemark:latest",
 					"env": [
 						{
 							"name": "CONTENT_TYPE",
```

Comment:
However, if we want to experiment with different images, we can push them to {{registry}}/{{project}} and use them within the project. So probably this is something we'd want to do in future.